src/__tests__/__fixtures__/schema.json

```diff
@@ -127,6 +127,37 @@
             "isDeprecated": false,
             "deprecationReason": null
           },
+          {
+            "name": "listNestedList",
```

Comment:
Looks like this may be missing the corresponding change to `src/__tests__/__fixtures__/schema.ts`:
```
    listNestedList: [[Nested!]!]!
```